### DISCUSSION TITLE
Compatible with Brother Innov'is NV880e ?

### DISCUSSION THREAD

**Discussion by G2brabant** [inkstitch/inkstitch#1896] [Q&A]
Hi, 

I'm about to purchase the Brother Innov'is NV880e but won't be able to afford the softwares Brother recommend, hence I discovered Inkstitch :) Thanks a lot for giving access to a free software. I just want to make sure Inkstitch will be compatible with this machine and I'll be able to make embroideries out of my graphic designs. Can you help by any chance?

Thanks a lot,
Best,
Guillemette

**Comment by tyrosinase** [comment]
Yup, Ink/Stitch produces PES, DST, and PEC, all of which are formats a modern Brother can read (though I always use just PES on mine).

There is one caveat: PES will take TRIM and I think STOP commands but that doesn't mean your machine will recognize them; my PE800 will only stop/trim on color changes. So Ink/Stitch won't warn you that those won't work because that's up to your machine.

**Reply by G2brabant** [reply]
Sounds perfect, thank you so much! :)
Cheers
